ClimateMonitorV2/views/index.pug

```diff
@@ -10,19 +10,21 @@ block content
     script(src="https://cdn.jsdelivr.net/npm/chartjs-plugin-zoom@0.7.7")
     script(src="https://cdn.jsdelivr.net/npm/litepicker/dist/js/main.js")
 
-    header
-        a(class="page-header") SheffSense: Sheffield Climate Monitor
     
     body
 
         div(class="sidebar")
-            img(class="img-fluid", id="headerimage", src="sheffsense.png") 
+            div(class="sidebar-header")
+                a(href="https://github.com/dambem/ClimateMonitorV2", id="sheffsenseImageHolder", target="_blank")
+                    img(class="img-fluid", id="sheffsenseImage", src="sheffsense.png")
+
             a(class="nav-link-active" href="/") Home
             a(class="nav-link" href="/about") About
             a(class="nav-link" href="/detailedstats") Detailed Statistics
             div(class="sidebar-footer")
                 a(href="https://github.com/dambem/ClimateMonitorV2" target="_blank")
```

Comment:
The href of the image points to the github (same as the sheffsense logo). We shouldn't need two images in the sidebar pointing links to the same location

Reply:
Fixed in latest commit now